backends/samsung/CMakeLists.txt

```diff
@@ -0,0 +1,141 @@
+
+    set(_enn_schema_generate_dir "${CMAKE_BINARY_DIR}/schema/include/executorch/backends/samsung")
+    # Paths to headers generated from the .fbs files.
+    string(REGEX REPLACE "serialization/([^/]+)[.]fbs$" "\\1_generated.h" generated_header "${fbs_file}")
+    set(_enn_schema_output "${_enn_schema_generate_dir}/${generated_header}")
+
+    # Generate the headers from the .fbs files.
+    add_custom_command(
+        OUTPUT ${_enn_schema_output}
+        COMMAND
+        ${FLATC_EXECUTABLE} --cpp --cpp-std c++11 --scoped-enums -o
+        "${_enn_schema_generate_dir}"
+        ${_enn_compile_options_schema}
+        DEPENDS ${_enn_compile_options_schema}
+        WORKING_DIRECTORY ${EXECUTORCH_SOURCE_DIR}
+        COMMENT "Generating enn compile options headers"
+        VERBATIM
+    )
+    add_custom_target(enn_compile_options_output ALL DEPENDS ${_enn_schema_output})
+
+    set_target_properties(PyEnnWrapperAdaptor PROPERTIES CXX_VISIBILITY_PRESET hidden)
+    target_link_libraries(PyEnnWrapperAdaptor PRIVATE
+        pybind11::module
+        pybind11::lto
+        graphgen_api
+        enn_logging
+    )
+    target_include_directories(PyEnnWrapperAdaptor BEFORE PRIVATE
+        ${CMAKE_BINARY_DIR}/schema/include
+        ${EXECUTORCH_SOURCE_DIR}/third-party/flatbuffers/include
+    )
+    add_dependencies(PyEnnWrapperAdaptor enn_compile_options_output)
+    pybind11_extension(PyEnnWrapperAdaptor)
+
+    # PyGraphWrapperAdaptor
+    add_library(PyGraphWrapperAdaptor MODULE)
+    #
+    find_library(GRAPH_WRAPPER_LIB NAMES graph_wrapper HINTS ${EXYNOS_AI_LITECORE_PATH}/lib64/)
+    add_library(graph_wrapper SHARED IMPORTED GLOBAL)
+    set_target_properties(graph_wrapper PROPERTIES
+            INTERFACE_INCLUDE_DIRECTORIES "${EXYNOS_AI_LITECORE_PATH}/include"
+            IMPORTED_LOCATION "${GRAPH_WRAPPER_LIB}"
+    )
+    set_target_properties(PyGraphWrapperAdaptor PROPERTIES CXX_VISIBILITY_PRESET hidden)
+    target_link_libraries(PyGraphWrapperAdaptor PRIVATE
+            pybind11::module
+            pybind11::lto
+            graph_wrapper
+            enn_logging
+    )
+    pybind11_extension(PyGraphWrapperAdaptor)
+
+    add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/aot)
+endif()
+
+
+if(${ANDROID})
+    target_link_libraries(enn_logging PRIVATE ${android_log})
+    add_library(enn_backend STATIC)
+    target_link_libraries(enn_backend PRIVATE enn_logging)
+    target_link_options_shared_lib(enn_backend)
+
+    set(__enn_executor_runner_srcs ${EXECUTORCH_SOURCE_DIR}/examples/samsung/executor_runner/enn_executor_runner.cpp)
+    add_executable(enn_executor_runner ${__enn_executor_runner_srcs})
+    target_link_libraries(enn_executor_runner PRIVATE
+            enn_logging
+            enn_backend
+            gflags
+            executorch
+            extension_data_loader
+            portable_ops_lib
+    )
+    set_target_properties(enn_executor_runner PROPERTIES CXX_VISIBILITY_PRESET hidden)
+    install(
```

Comment:
To make this work with CMake's find_package functionality, you can add an EXPORT directive here. See https://github.com/pytorch/executorch/blob/d760b772034bfedf49d9fefed2911e63449af623/backends/xnnpack/CMakeLists.txt#L124, for an example. This will also be necessary to add this backend target to the executorch_backends target.